### PR TITLE
Replace eslint-plugin-node to eslint-plugin-n.

### PR DESCRIPTION
**Pull request by heyyou3**
# Context

- eslint-plugin-node project has been no longer maintained.
- Use its successor, [eslint-plugin-n](https://github.com/eslint-community/eslint-plugin-n).
- renovate now recommends Replace.

image

# Check List

- [x] `eslint-plugin-node` rule compatibility.
    - https://www.npmjs.com/package/eslint-plugin-n

### CODE REVIEW

File: node/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Change log
 
+## v0.3.1
+
+### Non-breaking changes
```
Comment (heyyou3) [inline node/CHANGELOG.md]: [🤔 ] breaking changes…？

Comment (ryota-ka) [reply node/CHANGELOG.md]: It should be treated as a breaking change, as it is mandatory for the users of this package to replace their peer dependency.
Could you move this to the (newly-created) v0.4.0 section?

File: node/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+- Replace [eslint-plugin-node](https://github.com/mysticatea/eslint-plugin-node) to [eslint-plugin-n](https://github.com/eslint-community/eslint-plugin-n)
```
Comment (ryota-ka) [inline node/CHANGELOG.md]: npmjs.com should be more preferable as it's the very place they reside in

```suggestion
- Replace [eslint-plugin-node](https://www.npmjs.com/package/eslint-plugin-node) to [eslint-plugin-n](https://www.npmjs.com/package/eslint-plugin-n)
```

File: node/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@herp-inc/eslint-config-node",
-  "version": "0.3.0",
+  "version": "0.3.1",
```
Comment (ryota-ka) [inline node/package.json]: The `version` property will be automatically updated when the maintainer runs `$ yarn version` before publishing the package, so please keep it unchanged.

```suggestion
  "version": "0.3.0",
```

File: node/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
+  - eslint-plugin-node project has been no longer maintained.
+  - Use its successor, [eslint-plugin-n](https://github.com/eslint-community/eslint-plugin-n).
```
Comment (ryota-ka) [inline node/CHANGELOG.md]: This part is already a part of this change log itself.

```suggestion
```

File: node/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,13 @@
 # Change log
 
+## v0.3.1
+
+### Non-breaking changes
+
+- Replace [eslint-plugin-node](https://github.com/mysticatea/eslint-plugin-node) to [eslint-plugin-n](https://github.com/eslint-community/eslint-plugin-n)
+  - eslint-plugin-node project has been no longer maintained.
```
Comment (ryota-ka) [inline node/CHANGELOG.md]: Please quote the package names for the consistency.

```suggestion
- Replace [`eslint-plugin-node`](https://github.com/mysticatea/eslint-plugin-node) to [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n)
  - `eslint-plugin-node` project has been no longer maintained.
```

**Review by ryota-ka — CHANGES_REQUESTED** [review]
Thank you for your contribution!

I've added some request changes.
Could you please check them out?

**Comment by heyyou3** [comment]
@ryota-ka 
Thx for your confirmation 😍 
I have made the corrections, so please re-check!

**Review by ryota-ka — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**